_posts/2022-11-21-november-demo-day.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: Dask Demo Day November 2022
+author: Richard Pelgrim (Coiled)
+tagline: an optional tagline
+tags: [Community]
+theme: twitter
+draft: true
+---
+
+{% include JB/setup %}
+
+# Dask Demo Day : November
+
+Once a month, the Dask Community team hosts Dask Demo Day: an informal and fun online hangout where folks can showcase new or lesser-known Dask features…and the rest of us can learn about all the things we didn’t know Dask could do 😁
```

Comment:
Can we link "Dask Demo Day" to something? Maybe the calendar page or the community issue?

Reply:
```suggestion
Once a month, the Dask Community team hosts Dask Demo Day: an informal and fun online hangout where folks can showcase new or lesser-known Dask features and the rest of us can learn about all the things we didn’t know Dask could do 😁
```